source/CCustomOpcodeSystem.cpp

```diff
@@ -170,6 +169,10 @@ namespace CLEO {
 		ss << thread->GetName() << " opcode " << opcodeToStr(opcode) << std::endl;
 		OutputDebugStringA(ss.str().c_str());//*/
 
+		lastScript = thread;
+		lastOpcode = opcode;
+		lastOpcodePtr = (WORD*)thread->GetBytePointer() - 1; // rewind to the opcode start

```

Comment:
why -1? opcode id is 2 bytes

Reply:
this is WORD pointer, arithmetics take type size into account

Reply:
ah, I see. one word back